src/udev/scsi_id/scsi_id.c

```diff
@@ -84,6 +84,9 @@ static void set_type(unsigned type_num, char *to, size_t len) {
         case 0xf:
                 type = "optical";
                 break;
+        case 0x14:
+                type = "zbc";
```

Comment:
ZBC is abbreviation of Zoned Block Command. That sounds strange for me, as command is not type.
How about `"host managed zoned block"` ?
https://github.com/hreinecke/sg3_utils/blob/516129c21d8aa574e3075e908477ca8f5db0fd98/lib/sg_lib_data.c#L1600

Reply:
Uh, so `zbc` is consistent with `lsscsi`...
https://github.com/hreinecke/lsscsi/blob/8d34e7ddbabc69e04ae5b28dc14aa3a448cc3201/src/lsscsi.c#L194

Reply:
OK. So, feel free to keep this type as `zbc`.